r/R/convert-array.R

```diff
@@ -49,6 +49,12 @@
 #' - [character()]: String and large string types can be converted to
 #'   [character()]. The conversion does not check for valid UTF-8: if you need
 #'   finer-grained control over encodings, use `to = blob::blob()`.
+#' - [factor()]: Dictionary-encoded arrays of strings can be converted to
+#'   [factor()]; however, this must be specified explicitly (i.e.,
```

Comment:
Nit: do you need two links? It will be two links in the pkgdown docs anyway, though.

```suggestion
#'   `factor()`; however, this must be specified explicitly (i.e.,
```